saleor/graphql/schema.graphql

```diff
@@ -2407,6 +2471,12 @@ type Mutation {
   permissionGroupDelete(id: ID!): PermissionGroupDelete
   permissionGroupAssignUsers(id: ID!, users: [ID!]!): PermissionGroupAssignUsers
   permissionGroupUnassignUsers(id: ID!, users: [ID!]!): PermissionGroupUnassignUsers
+  invoiceCreate(orderID: ID!, input: InvoiceInput): InvoiceCreate
+  invoiceDelete(orderID: ID!, invoiceID: ID!): InvoiceDelete
+  invoiceUpdate(invoiceID: ID!, input: InvoiceInput): InvoiceUpdate
+  invoiceSend(id: ID!, email: String): InvoiceSend
+  requestInvoiceCreate(orderID: ID!, input: RequestInvoiceCreateInput): RequestInvoiceCreate
+  requestInvoiceDelete(orderID: ID!): RequestInvoiceDelete
```

Comment:
Assuming, the order has 5 invoices. Which one do you want to remove here? :D

Reply:
Typo!